.github/workflows/generic-deb-workflow.yml

```diff
@@ -72,10 +72,24 @@ jobs:
     runs-on: ubuntu-22.04
     steps:
     - uses: actions/checkout@v4
+
     - uses: actions/setup-python@v2
       with:
         python-version: '3.10'
-    - uses: pre-commit/action@v2.0.3
+
+    - name: Check if pre-commit config exists
```

Comment:
better to test file existence manually and return the status in the github output

This actions is already deprecated and have the warning:
`Warning: The `set-output` command is deprecated and will be disabled soon.`

Reply:
I've bumped the action version, it no longer has the warning :eyes: